src/components/organisms/AssetActions/Pool/index.tsx

```diff
@@ -278,15 +278,15 @@ export default function Pool(): ReactElement {
             <Tooltip content={content.tooltips.price} />
             <div className={styles.dataTokenLinks}>
               <ExplorerLink
-                networkId={networkId}
+                networkId={ddo.chainId}
                 path={`address/${price?.address}`}
               >
                 Pool
               </ExplorerLink>
               <ExplorerLink
-                networkId={networkId}
+                networkId={ddo.chainId}
                 path={
-                  networkId === 137 || networkId === 1287
+                  ddo.chainId === 137 || ddo.chainId === 1287
```

Comment:
forgot about this check. We recently switched Polygon to use `polygonscan.com` as explorer in ocean.js ConfigHelper, which behaves just like etherscan. So we can remove the check for `137` as it's now `token` there too, e.g. https://polygonscan.com/token/0x282d8efce846a88b159800bd4130ad77443fa1a1

In fact all explorer links we now have in ConfigHelper behave like etherscan, but still 2 left we need as check: `1287` & `2021000`

Reply:
Right, I checked Polygon and Mainnet and no need for check. Not sure what `2021000` is

Reply:
I have this file bookmarked just for that cause I never can remember chainIds :-) https://github.com/oceanprotocol/ocean.js/blob/main/src/utils/ConfigHelper.ts

`2021000` is Gaia-X testnet but so far we are not dealing with it but we can still add the check for it here as its expected this will always stay blockscout-based explorer so will need `/tokens/` here

Reply:
Cool, thanks